tasmota/tasmota_xsns_sensor/xsns_100_ina3221.ino

```diff
@@ -70,45 +97,41 @@
 #define INA3221_ENABLE_MASK                     (0x7000)
 #define INA3221_ENABLE_CH(ch)                   (0x4000>>(ch))    // default: set
 
-#define INA3221_AVERAGING_MASK                  (0x0E00)
-#define INA3221_AVERAGING_NONE                  (0x0000)  // 1 sample, default
+#define INA3221_AVERAGING_NONE                  (0x0000)
 #define INA3221_AVERAGING_4_SAMPLES             (0x0200)
 #define INA3221_AVERAGING_16_SAMPLES            (0x0400)
 #define INA3221_AVERAGING_64_SAMPLES            (0x0600)
 #define INA3221_AVERAGING_128_SAMPLES           (0x0800)
 #define INA3221_AVERAGING_256_SAMPLES           (0x0A00)
-#define INA3221_AVERAGING_512_SAMPLES           (0x0C00)
+#define INA3221_AVERAGING_512_SAMPLES           (0x0C00)  // 1 sample, default
 #define INA3221_AVERAGING_1024_SAMPLES          (0x0E00)
 
-#define INA3221_VBUS_CONV_TIME_MASK             (0x01C0)
 #define INA3221_VBUS_CONV_TIME_140US            (0x0000)
 #define INA3221_VBUS_CONV_TIME_204US            (0x0040)
 #define INA3221_VBUS_CONV_TIME_332US            (0x0080)
 #define INA3221_VBUS_CONV_TIME_588US            (0x00C0)
-#define INA3221_VBUS_CONV_TIME_1MS              (0x0100)  // 1.1ms, default
+#define INA3221_VBUS_CONV_TIME_1MS              (0x0100)  // 1.1ms
 #define INA3221_VBUS_CONV_TIME_2MS              (0x0140)  // 2.116ms
 #define INA3221_VBUS_CONV_TIME_4MS              (0x0180)  // 4.156ms
-#define INA3221_VBUS_CONV_TIME_8MS              (0x01C0)  // 8.244ms
+#define INA3221_VBUS_CONV_TIME_8MS              (0x01C0)  // 8.244ms, default
 
-#define INA3221_SHUNT_CONV_TIME_MASK            (0x0038)
 #define INA3221_SHUNT_CONV_TIME_140US           (0x0000)
 #define INA3221_SHUNT_CONV_TIME_204US           (0x0008)
 #define INA3221_SHUNT_CONV_TIME_332US           (0x0010)
 #define INA3221_SHUNT_CONV_TIME_588US           (0x0018)
-#define INA3221_SHUNT_CONV_TIME_1MS             (0x0020)  // 1.1ms, default
+#define INA3221_SHUNT_CONV_TIME_1MS             (0x0020)  // 1.1ms
 #define INA3221_SHUNT_CONV_TIME_2MS             (0x0028)  // 2.116ms
-#define INA3221_SHUNT_CONV_TIME_4MS             (0x0030)  // 4.156ms
+#define INA3221_SHUNT_CONV_TIME_4MS             (0x0030)  // 4.156ms, default
 #define INA3221_SHUNT_CONV_TIME_8MS             (0x0038)  // 8.244ms
 
-#define INA3221_MODE_MASK                       (0x0007)
```

Comment:
Really ?

Reply:
ok that was my setting ..... but it works without problems - we read the registers once a second - so there shoud bee enough time for the chip and we have a better filtering